forge-gui/res/editions/The Lost Caverns of Ixalan.txt

```diff
@@ -3,6 +3,10 @@ Code=LCI
 Date=2023-11-17
 Name=The Lost Caverns of Ixalan
 Type=Expansion
+BoosterCovers=3
+Booster=9 Common:!dfc:fromsheet("LCI cards"), 3 Uncommon:!dfc:fromSheet("LCI cards"), 1 fromSheet("LCI double-face"), 1 RareMythic:fromSheet("LCI cards"), 1 BasicLand:fromSheet("LCI cards")
```

Comment:
Is this putting a Cave in 70% of the packs? Not sure Cave counts as a basic?

My guess is this might need another weighted list like what you did for the C/U double-faced

Reply:
My bad, I completely missed that it said cave :D

Reply:
There seems to be 5 cards where the backside is a cave. I assume they aren't included in this slot?

https://scryfall.com/search?q=set%3Alci+type%3Acave&unique=cards&as=grid&order=name

Reply:
Yea let's assume no. They aren't really "Lands that are caves or Lands that are full art" they are DFC yadayada

Reply:
Sorry I deleted that comment because from what I can see from people opening packs it is specifically the caves that have L rarity, so I went with that